resources/storage_blob_services.go

```diff
@@ -0,0 +1,233 @@
+			},
+			{
+				Name:        "container_delete_retention_policy_days",
+				Description: "Indicates the number of days that the deleted item should be retained.",
+				Type:        schema.TypeInt,
+				Resolver:    schema.PathResolver("BlobServicePropertiesProperties.ContainerDeleteRetentionPolicy.Days"),
+			},
+			{
+				Name:        "last_access_time_tracking_policy_enable",
+				Description: "When set to true last access time based tracking is enabled",
+				Type:        schema.TypeBool,
+				Resolver:    schema.PathResolver("BlobServicePropertiesProperties.LastAccessTimeTrackingPolicy.Enable"),
+			},
+			{
+				Name:        "last_access_time_tracking_policy_name",
+				Description: "Name of the policy.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("BlobServicePropertiesProperties.LastAccessTimeTrackingPolicy.Name"),
+			},
+			{
+				Name:        "last_access_time_tracking_policy_tracking_granularity_in_days",
+				Description: "The field specifies blob object tracking granularity in days.",
+				Type:        schema.TypeInt,
+				Resolver:    schema.PathResolver("BlobServicePropertiesProperties.LastAccessTimeTrackingPolicy.TrackingGranularityInDays"),
+			},
+			{
+				Name:        "last_access_time_tracking_policy_blob_type",
+				Description: "An array of predefined supported blob types.",
+				Type:        schema.TypeStringArray,
+				Resolver:    schema.PathResolver("BlobServicePropertiesProperties.LastAccessTimeTrackingPolicy.BlobType"),
+			},
+			{
+				Name:        "sku_name",
+				Description: "Possible values include: 'StandardLRS', 'StandardGRS', 'StandardRAGRS', 'StandardZRS', 'PremiumLRS', 'PremiumZRS', 'StandardGZRS', 'StandardRAGZRS'",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Sku.Name"),
+			},
+			{
+				Name:        "sku_tier",
+				Description: "Possible values include: 'Standard', 'Premium'",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("Sku.Tier"),
+			},
+			{
+				Name:        "id",
+				Description: "Fully qualified resource ID.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ID"),
+			},
+			{
+				Name:        "name",
+				Description: "The name of the resource",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "type",
+				Description: "The type of the resource.",
+				Type:        schema.TypeString,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "azure_storage_blob_service_cors_rules",
+				Description: "CorsRule specifies a CORS rule for the Blob service",
+				Resolver:    fetchStorageBlobServiceCorsRules,
+				Columns: []schema.Column{
+					{
+						Name:        "blob_service_cq_id",
+						Description: "Unique CloudQuery ID of azure_storage_blob_services table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
```

Comment:
Would add parent field resolver for blob_id

Reply:
Done